### PR TITLE
PREAPPS-4641 Propose New Time for meetings -- Part1

### PR DESCRIPTION
**Pull request by damini-vashishtha**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2106,7 +2106,7 @@
     },
     "@types/valid-url": {
       "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/@types/valid-url/-/valid-url-1.0.2.tgz",
+      "resolved": "http://registry.npmjs.org/@types/valid-url/-/valid-url-1.0.2.tgz",
```
Comment (silentsakky) [inline package-lock.json:2109]: not needed changes

File: src/schema/schema.graphql [hunk 2]
```diff
@@ -737,6 +739,11 @@ type TzOnsetInfo { # <standard>, <daylight>
 	sec: Int
 }
 
+type AppointmentInfo { # <appt>; aka AppointmentHitInfo
+id: ID!
```
Comment (silentsakky) [inline src/schema/schema.graphql:743]: indentation

Comment (damini-vashishtha) [reply src/schema/schema.graphql]: Done

File: package-lock.json (hunk 1 above)
Comment (damini-vashishtha) [reply package-lock.json]: Done.

Comment (BhavinBathani) [reply package-lock.json]: This is still showing some permission-mod related changes like "100644 → 100755". Can't we get rid of that as well.

File: src/schema/schema.graphql [hunk 3]
```diff
@@ -737,6 +739,11 @@ type TzOnsetInfo { # <standard>, <daylight>
 	sec: Int
 }
 
+type AppointmentInfo { # <appt>; aka AppointmentHitInfo
```
Comment (silentsakky) [inline src/schema/schema.graphql:742]: how is this different then CalendarItemHitInfo ?

Comment (silentsakky) [reply src/schema/schema.graphql:742]: both contains same data

Comment (damini-vashishtha) [reply src/schema/schema.graphql:742]: CalendarItemHitInfo having so many mandatory fields, which is not the case here

Comment (silentsakky) [reply src/schema/schema.graphql:742]: ok

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@zimbra/api-client",
 	"amdName": "zmApiJsClient",
-	"version": "36.1.0",
+	"version": "36.2.0",
```
Comment (silentsakky) [inline package.json:4]: why do we need this?

Comment (damini-vashishtha) [reply package.json:4]: I am not sure.I just did rebase and It came from there. In blame, showing  Yash wrote this.
In masters also its showing the same version

**Comment by silentsakky** [comment]
Changes moved to https://github.com/Zimbra/zm-api-js-client/pull/359

**Pull request closed without merging**